src/lib/pbkdf/argon2/argon2.cpp

```diff
@@ -21,6 +21,8 @@
    #include <botan/internal/cpuid.h>
 #endif
 
+#include <limits>
```

Comment:
Include this in the main block of `#include` rather than after the conditionals

Reply:
Fixed